### ISSUE TITLE
Menu visual bug

### ISSUE THREAD

**Issue by kacernator** [exaile/exaile#540]
Switch left panel to "collection". When moving mouse over horizontal line separator then text has visual bug. 

Version:
4.0.0beta3
GStreamer: 1.12.5
GTK+: 3.22.30
Locale: cs_CZ UTF-8
Mutagen: 1.40.0
PyGObject: 3.26.1
Python: 2.7.14

Link to video of the bug:
https://youtu.be/yRkJKOyvNMs

**Comment by sjohannes** [comment]
This bug happens when the GTK+ theme is missing support for GtkTreeView separators. Known themes with this issue (some have been fixed):

* [Ambiance/Radiance](https://bugs.launchpad.net/ubuntu-themes/+bug/1701512)
* [Box](https://bugs.launchpad.net/lubuntu-artwork/+bug/1781689)
* [Breeze](https://bugs.kde.org/show_bug.cgi?id=380291)

Which theme are you using? If you're using a different one, it would be great if you could report this issue to the theme author.

(Closing as duplicate of https://github.com/exaile/exaile/issues/362 )

**Comment by kacernator** [comment]
I am using Breeze theme for GTK3. So what should I do? I dont want to set different theme for GTK3 apps.

**Comment by sjohannes** [comment]
I'm not sure; the Breeze [bug report](https://bugs.kde.org/show_bug.cgi?id=380291) doesn't have any activity. Perhaps it's worth contacting the developers through another way (mailing list / IRC)? You could also try filing a bug report to your OS, maybe the package maintainer can help fixing it.

If none of that works, you can add this to `settings.ini` to disable the treeview separators:

```ini
[gui]
draw_separators = B: False
```

but it would be better to get this fixed properly in the theme.

**Comment by sjohannes** [comment]
Oh, and if you're interested in trying to fix this yourself, the Ambiance/Radiance [patch](https://code.launchpad.net/~c-lobrano/ubuntu-themes/fix-1701512/+merge/327057/+preview-diff/801581/+files/preview.diff) might be helpful. I think it's mostly just adding this:

```css
treeview.view.separator {
    min-height: 2px;
}
```

In fact, you could even try adding it to your user `gtk.css` and see if that fixes the issue.

**Comment by sjohannes** [comment]
The separators make the list easier to read and helps when looking for particular names. I find the separators very useful and I think most users would agree. That's why I strongly encourage people to try to fix the themes, rather than disabling this feature.
